apps/publikator/lib/translations.json

```diff
@@ -1202,7 +1202,31 @@
     },
     {
       "key": "memo/modal/warning/newDoc",
-      "value": "Sie können erst Kommentare hinterlassen, nachdem Sie den Beitrag kommittet haben."
+      "value": "Sie können eine Notiz hinterlassen, nachdem Sie den Beitrag comitted haben."
```

Comment:
`comitted` with one `m`?

Reply:
That's a typo. Adressed in https://github.com/republik/plattform/pull/3/commits/5d1529ed28387d9a766752ec5b86b7ad1f0bc5fe